models/uniswap/arbitrum/uniswap_arbitrum_schema.yml

```diff
@@ -0,0 +1,132 @@
+version: 2
+
+models:
+  - name: uniswap_v3_arbitrum_trades
+    meta:
+      blockchain: arbitrum
+      sector: dex
+      project: uniswap_v3
+      contributors: jeff-dude, masquot, soispoke, mtitus6
+    config:
+      tags: ['arbitrum','uniswap_v3','trades', 'uniswap','dex']
+    description: >
+        Uniswap V3 contract trades on Arbitrum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
+      - &project 
+        name: project
+        description: "Project name of the DEX"  
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DEX project"
+      - &block_date
+        name: block_date
+        description: "UTC event block date of each DEX trade"
+      - &block_time
+        name: block_time
+        description: "UTC event block time of each DEX trade"
+      - &token_bought_symbol
+        name: token_bought_symbol
+        description: "Token symbol for token bought in the transaction"
+      - &token_sold_symbol
+        name: token_sold_symbol
+        description: "Token symbol for token sold in the transaction"
+      - &token_pair
+        name: token_pair
+        description: "Token symbol pair for each token involved in the transaction"
+      - &token_bought_amount
+        name: token_bought_amount
+        description: "Value of the token bought at time of execution in the original currency"
+      - &token_sold_amount
+        name: token_sold_amount
+        description: "Value of the token sold at time of execution in the original currency"
+      - &token_bought_amount_raw
+        name: token_bought_amount_raw
+        description: "Raw value of the token bought at time of execution in the original currency"
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution in the original currency"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+      - &token_bought_address
+        name: token_bought_address
+        description: "Contract address of the token bought"
+        tests:
```

Comment:
in order for this test to run, you need to manually add rows to our dex trades seed file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

uniswap has a linked analytics page on their app (that's how i did for ethereum version), you can find example tx's there where you can open on etherscan and fill out rows as needed

the goal is to ensure the columns pulled for bought/sold are applied correctly & not flipped (common issue on v1 engine)

Reply:
I added 3 transactions to the seed file. USDC to WETH , WETH to USDC, and USDC to DAI